python/tk_framework_desktopserver/shotgun/scripts/execute_command.py

```diff
@@ -13,10 +13,22 @@
 import os
 
 def execute(config, project, name, entities, base_configuration, engine_name):
+    """
+    Executes an engine command in the desired environment.
+
+    :param dict config: The pipeline configuration entity.
+    :param dict project: The project entity.
+    :param list entities: The list of entities selected in the web UI when the
+        command action was triggered.
+    :param str base_configuration: The desired base pipeline configuration's
+        uri.
+    :param str engine_name: The name of the engine to bootstrap into. This
+        is most likely going to be "tk-shotgun"
+    """
+    # The local import of sgtk ensures that it occurs after sys.path is set
+    # to what the server sent over.
     import sgtk
```

Comment:
Thanks.

That variable isn't related, no. It was the allow_config_overrides property that I added to ToolkitManager.